queries/ocamllex/highlights.scm

```diff
@@ -16,7 +18,7 @@
 (regexp_repetition ["*"] @operator)
 (regexp_strict_repetition ["+"] @operator)
 
-(action ["{" "}"] @punctuation.special) @embedded
+(action ["{" "}"] @punctuation.special)
```

Comment:
What is this action? Might `@none` apply here to reset the highlighting or is this always never highlighted.

Reply:
action contains only the brackets and the ocaml node, so resetting the highlighting for the ocaml node is enough, I think.